component/src/main/java/io/siddhi/extension/util/Constant.java

```diff
@@ -30,4 +30,8 @@
     public static final String SFTP_SCHEME_FILE_OPTION = "sftp";
     public static final String VFS_SCHEME_KEY = "VFS_SCHEME";
     public static final String FTP_SCHEME_FILE_OPTION = "ftp";
+    public static final String STATUS_DONE = "Completed";
+    public static final String STATUS_PROCESS = "Copying";
```

Comment:
since the method is "onChange" lets name this as "modifying"

Reply:
fixed in 3f9a2a0